addon/templates/components/table.hbs

```diff
@@ -51,26 +51,22 @@
                         <EmberTh::ResizeHandle @columnMeta={{columnMeta}} />
                     {{/if}}
                     {{#if (and this.canPanLeft (eq columnValue this.firstVisibleColumn))}}
-                        <ButtonBasic
-                            @type="secondary"
-                            @size="sm"
-                            @slim={{true}}
-                            @class="data-table-col-pan-btn data-table-col-pan-btn-left"
-                            @action={{fn this.panColumnsLeft}}
+                        <button
+                            class="data-table-col-pan-btn data-table-col-pan-btn-left {{@buttonClass}}"
```

Comment:
@napafundi i might rename this to be a bit more specific, maybe `@panButtonClass`. Also might be a good idea to just default the value in the table component, e.g. to `btn btn-secondary`, for ease of use.